core-services/prow/02_config/_config.yaml

```diff
@@ -2465,6 +2462,54 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - operator-framework/operator-lib
+  - includedBranches:
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    labels:
+    - bugzilla/valid-bug
+    - cherry-pick-approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/origin
+  - includedBranches:
+    - release-4.6
+    labels:
+    - group-lead-approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    repos:
+    - openshift/origin
+  - excludedBranches:
+    - release-3.11
```

Comment:
I looked at the `release-3.11` query and it has the same criteria like this query has, so to make the queries more independent, I would prefer to remove `openshift/origin` from that query, and remove this `release-3.11` line from here, for the `rleesae-3.11` branch to be covered by this query.

Reply:
got it, done